### ISSUE TITLE
Improved Text editor is broken and uncanny

### ISSUE THREAD

**Issue by jerome-poenisch** [brcontainer/stack-exchange-notifications#26]
I like the option for the improved text editor and having the side-by-side view of markdown and rendered really helps.

For some reason when I add a new answer everything works fine and I can hit "Save" and it posts my answer correctly. **However**, when I edit an already existing post, no matter if mine, of someone else, a question o answer .. I can't "Save". So currently I always end up editing a post, select everything hit CTRL+C. Then I press EXCAPE in order to leave the editor and then again press edit and paste in the copied text. 

Also it is a little bit uncanny that the "improved editor" is automatically full screen -> You can't see the question anymore you are currently trying to answer.

So TL;DR:
It would be great if 
 - the functionality o the Save button itself worked again
 - you could still see the Question in order to e.g. copy content of it and refer to it while answering.

**Comment by jerome-poenisch** [comment]
This seems in particular related to the FullScreen options

**Comment by vanowm** [comment]
Not only it doesn't seem possible exit fullscreen editor, but "improved" non-fullscreen editor doesn't show preview, making it useless.
